### PR TITLE
Fix #946 by adding a message

### LINKED ISSUE #946: Add message stating that instructor training applications for Fall 2016 are closed
Please add a prominent message to the instructor training application form saying:

~~~
Applications for the open Fall 2016 online instructor training class are now closed,
but we are still accepting applications for subsequent courses.
If you are scheduled to receive training at a partner site,
and have been asked to fill in this form to give us information about yourself,
please do so.
~~~

### PR DESCRIPTION
**Pull request by pbanaszkiewicz**
Fall 2016 open-access training is closed, but we're taking requests for
the next one.

Preview:
screen shot 2016-08-09 at 17 00 51

### CODE REVIEW

File: workshops/templates/forms/trainingrequest.html [hunk 1]
```diff
@@ -57,6 +58,11 @@
 <p>For more information on Software and Data Carpentry instructor training,
 please see <a href="http://swcarpentry.github.io/instructor-training/" target="_blank">
 http://swcarpentry.github.io/instructor-training/</a>.</p>
+
+<div class="alert alert-warning">
+  <p><strong>Warning:</strong> Applications for the open Fall 2016 online instructor training class are now <strong>closed</strong>, but we are still accepting applications for subsequent courses. If you are scheduled to receive training at a partner site, and have been asked to fill in this form to give us information about yourself, please do so.</p>
```
Comment (gvwilson) [inline workshops/templates/forms/trainingrequest.html]: 1. Review would be easier if you line-wrapped :-)
2. Please remove the word "Warning", and the bold around "closed".
3. The rest looks good to me - @maneesha, can you please review the wording?

**Comment by pbanaszkiewicz** [comment]
@gvwilson @maneesha fixed

**Comment by gvwilson** [comment]
Looks good to me - please merge as soon as @maneesha gives it a thumbs-up.

**Comment by pbanaszkiewicz** [comment]
Ping @maneesha

**Pull request merged**